src/oscar/apps/order/config.py

```diff
@@ -6,3 +6,7 @@ class OrderConfig(AppConfig):
     label = 'order'
     name = 'oscar.apps.order'
     verbose_name = _('Order')
+
+    def ready(self):
+        from . import receivers  # noqa
+        super(OrderConfig, self).ready()
```

Comment:
I don't think we need a `super()` call here - the parent method is a noop.

Reply:
Ok, I just do that for good practice but I'll remove it